libs/cardano-data/default.nix

```
@@ -0,0 +1,4 @@
+let
+  pkgs = import ../../../default.nix {};
+in
+  pkgs.libs.small-steps
```

Looks like we missed some
```suggestion
  pkgs.libs.cardano-data
```

---

done